lighthouse-core/computed/metrics/largest-contentful-paint.js

```diff
@@ -5,26 +5,27 @@
  */
 'use strict';
 
-const makeComputedArtifact = require('../computed-artifact.js');
-const ComputedMetric = require('./metric.js');
-const LHError = require('../../lib/lh-error.js');
-
 /**
  * @fileoverview Computed Largest Contentful Paint (LCP), the paint time of the largest in-viewport contentful element
  * COMPAT: LCP's trace event was first introduced in m78. We can't surface an LCP for older Chrome versions
  * @see https://github.com/WICG/largest-contentful-paint
  * @see https://wicg.github.io/largest-contentful-paint/
  * @see https://web.dev/largest-contentful-paint
  */
+
+const makeComputedArtifact = require('../computed-artifact.js');
```

Comment:
just curious, any reason for moving these?

Reply:
I think a file overview comment  block should be the first thing in a file (except license / strict i guess)

do you hate it

Reply:
we're inconsistent about this already. it looks 50/50 in a brief survey.

being inconsistent would suggest that we leave existing ones as they lie until we fix all of them at once. (i think that's what we've done before in similar situations... require('xx.js').. right?)

that said i dont think its important this time. so lets ignore this one.. just this one time. :)